examples/stats/prometheus/main.go

```diff
@@ -75,7 +75,7 @@ func main() {
 		"processed video size over time",
 		nil,
 		videoSize,
-		view.DistributionAggregation([]float64{0, 1 << 16, 1 << 32}),
+		view.DistributionAggregation([]float64{1 << 10, 1<<15, 1 << 20, 1 << 25}),
```

Comment:
Can you send this changes in a different PR? They are not related to the bug fix.

Reply:
ACK, thanks.